fe/fe-core/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -150,6 +155,21 @@ public InsertStmt(InsertTarget target, String label, List<String> cols, InsertSo
                 && ((SelectStmt) queryStmt).getTableRefs().isEmpty());
     }
 
+    public InsertStmt(InsertTarget target, String label, List<String> cols, InsertSource source,
+             List<String> hints, boolean isPartialUpdate) {
+        this.tblName = target.getTblName();
```

Comment:
L160-L165 can be replaced with `this(target, label, cols, source, hints)?`

Reply:
fixed